source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/components/locale-auto-suggest.scss

```diff
@@ -0,0 +1,110 @@
+.phrase-with-highlight {
+  .highlight {
+    background: yellow;
+  }
+}
+
+
+.locale-input__has-match,
```

Comment:
a lot of this was copied from react-auto-suggest's demo. it was tweaked to allow for sizing / positioning, etc changes